Lib/asyncio/events.py

```diff
@@ -605,12 +605,8 @@ def new_event_loop(self):
 
 
 # A TLS for the running event loop, used by _get_running_loop.
-class _RunningLoop(threading.local):
-    _loop = None
-    _pid = None
-
-
-_running_loop = _RunningLoop()
+_running_loop = threading.local()
+_running_loop.loop_pid = (None, None)
 
```

Comment:
This change needs to be reverted to:

```python
class _RunningLoop(threading.local):
    _loop_pid = (None, None)
```

The problem is that `_running_loop.loop_pid = (None, None)` initializes `_running_loop.loop_pid` only for the *current* thread. All other threads using `_running_loop.loop_pid` will see `None`.